cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -280,6 +280,11 @@ func (c *CStorVolumeReplicaController) cVRAddEventHandler(
 				"CStorVolumeReplica %v is already imported",
 				string(cVR.ObjectMeta.UID),
 			)
+			if err := volumereplica.UpdateReplicaID(cVR); err != nil {
```

Comment:
yes. I have added here if upgrade/user missed to set replicaID. It will check if replicaID is set or not, if not set then it will update else return `nil`.

Reply:
We are reconciling for that it is good sorry for the wrong review